reactor-netty-http/src/main/java/reactor/netty/http/client/HttpClientFinalizer.java

```diff
@@ -165,8 +165,6 @@ static void discard(HttpClientOperations c) {
 		}
 	}
 
-	static final Function<ChannelOperations<?, ?>, Publisher<ByteBuf>> contentReceiver = ChannelOperations::receive;
```

Comment:
I do agree, indeed, if we keep the constant (but change it to ChannelOperations::receiveObject), then the impact of the patch will be reduced and the patch will be centralized in single place.

Reply:
The reason why I deleted this constant is because I found that there are no other references to it. Do I need to restore it? Or change the constant to ChannelOperations::receiveObject type?

Reply:
It was used in `WebsocketFinalizer`

Reply:
Then I'll make some adjustments

Reply:
@manzhizhen , could you also please revert the change from WebsocketFinalizer ? (because now, there is no more changes done in this file) ? thank you.

Reply:
Okay, I'll restore it